markdown-link-check/action.yml

```diff
@@ -0,0 +1,89 @@
+# Copyright (c) 2025, NVIDIA CORPORATION.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'Markdown Link Check'
+description: 'Check markdown links'
+
+runs:
+  using: "composite"
+  steps:
+    - name: Run markdown link check
+      id: markdown-link-check
+      uses: tcort/github-action-markdown-link-check@v1
+      with:
+        max-depth: -1
+        use-verbose-mode: 'yes'
+        base-branch: 'gh-pages'
+    
+    - name: Summarize results
+      shell: bash
+      if: always()
+      run: |
+        has_errors=0
```

Comment:
nit: the step name looks like a mislead if using the first command, try a meaningful comment?
<img width="460" height="381" alt="image" src="https://github.com/user-attachments/assets/96349260-584d-446d-bdfc-5ce5d11c1ff3" />